atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -2123,6 +2127,53 @@ public void compactInternally(TableReference tableRef) {
         }
     }
 
+    @Override
+    public NodeAvailabilityStatus getNodeAvailabilityStatus() {
+        try {
```

Comment:
Yes, that makes more sense thinking about it again. I am planning to squash `EACH_QUORUM_AVAILABLE` and `LOCAL_QUORUM_AVAILABLE` to `QUORUM_AVAILABLE` and return quorum_available only when less than half RF nodes are down.